lib/utils/BvEvent.js

```diff
@@ -23,7 +23,9 @@ export class BvEvent {
         let defaultPrevented = false;
         // Recreate preventDefault method. One way setter.
         this.preventDefault = function preventDefault() {
-            defaultPrevented = true;
+            if (this.cancelable) {
```

Comment:
However, i think it would be right to have boolean props switchers initialized as false.